float64[] start
float64[] end
---
bool status
string error_msg
